geolib/models/dsettlement/templates/input.sli.j2

```diff
@@ -279,7 +279,7 @@ D-Settlement={{ version.d__settlement }}
 {{ calculation_options.precon_pressure_within_layer.value }} : Precon. pressure within a layer = {{ calculation_options.precon_pressure_within_layer.name }}
 {{ calculation_options.is_imaginary_surface.value }} : Imaginary surface = {{ calculation_options.is_imaginary_surface.name }}
 {% if calculation_options.imaginary_surface_layer is not none %}
```

Comment:
What happens when  calculation_options.imaginary_surface_layer is none? I guess the line will not be written, but then DSetllement can't read the file anymore.